app/components/mandatarissen/update-state.js

```diff
@@ -14,6 +14,7 @@ export default class MandatarissenUpdateState extends Component {
   @tracked bestuursorganenForFractie = [];
   @tracked rangorde = null;
   @tracked selectedReplacement = null;
+  @tracked replacementSelected;
```

Comment:
Fine for now but if we are going to align everything on multiple replacements, we should change this from a boolean to a list of ids so that we can also track if there are no changes

Reply:
The variable name is maybe a bit confusing, the idea was more if a change happens, set this variable to true, but now with this component you can only select a replacement. Will update the name.